WebKitBrowser/CMakeLists.txt

```diff
@@ -61,8 +61,12 @@ set(PLUGIN_WEBKITBROWSER_TRANSPARENT "false" CACHE STRING "Set transparency")
 set(PLUGIN_WEBKITBROWSER_STARTURL "about:blank" CACHE STRING "Initial URL for WebKitBrowser plugin")
 set(PLUGIN_WEBKITBROWSER_USERAGENT "" CACHE STRING "User agent string")
 set(PLUGIN_WEBKITBROWSER_MEMORYPROFILE "512m" CACHE STRING "Memory Profile")
-set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE_WEBPROCESSLIMIT "300" CACHE STRING "Memory Pressure Webprocess Limit")
-set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE_NETWORKPROCESSLIMIT "100" CACHE STRING "Memory Pressure Networkprocess Limit")
+set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE_WEBPROCESS_SETTINGS_LIMIT "300" CACHE STRING "WebkitBrower Memory Pressure Webprocess Limit")
+set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE_WEBPROCESS_SETTINGS_GPU_LIMIT "1000" CACHE STRING "WebkitBrower Memory Pressure Webprocess GPU Limit")
```

Comment:
I'd prefer to keep these GPU-related settings unset to avoid overriding values set in the container configuration. It also makes sense to keep unset since GPU file is not specified.

Reply:
Environment variables have higher precedence over these settings. So even if these values are set through .json file, env. variables will always take into account with high priority. 
But it is ok to remove default setting of gpu_limit from CMakeLists.